### ISSUE TITLE
the only function name i get is `[no mark]`

### ISSUE THREAD

**Issue by weisjohn** [ConradIrwin/async-profile#3]
code: 

``` javascript
var asyncProfile = require('async-profile');

function doStuff() {
    console.log('do')
    setTimeout(function stuff() {
        console.log('stuff')
    }, 1e3);
}

process.nextTick(function() {
    new asyncProfile();
    process.nextTick(doStuff);
});
```

``` bash
$ node index.js 
do
stuff
total: 1.880ms (in 1004.292ms real time, CPU load: 0.0, wait time: 1003.656ms)
0.728: 1.512ms  [no mark] (0.133)  
2.416: 0.082ms    [no mark] (0.000)  
1004.209: 0.275ms    [no mark] (0.080)  
1004.683: 0.011ms      [no mark] (0.000)  
```

**Comment by ConradIrwin** [comment]
This is probably a bad fix of #1. Will look into it this evening.

**Comment by weisjohn** [comment]
@ConradIrwin thanks!

**Comment by weisjohn** [comment]
fyi, I'm on `node -v` v0.10.31

**Comment by cleancoderocker** [comment]
What is the status of this? I'm experiencing the same.

**Comment by weisjohn** [comment]
@ConradIrwin if you're will to consider rewriting in regular JS, I can help with this.

**Comment by ConradIrwin** [comment]
Sure! It's only in coffee script because I was writing it in a hurry — the
code is not something I'm particularly proud of.

Conrad

**Comment by weisjohn** [comment]
@ConradIrwin would you lead the rewrite? I can't read CoffeeScript.

**Comment by ConradIrwin** [comment]
Released as 0.4.0, please let me know if you're still having issues.
